### ISSUE TITLE
When clicking anything in your inventory, chat floods with messages.

### ISSUE THREAD

**Issue by zix3r** [yummypasta/MinecraftManhunt#19]
## Describe the bug
When you click anything in your inventory, the chat floods with **You're not on a ManHunt team!**

## Potential Causes
Click any slot in your inventory when the game hasn't started yet, it will show the message in chat.

## Environment Info
**Plugin version:** 1.3.1

**Bukkit/Spigot server version:** 1.16.4

## Configuration file
Sending us your plugin `config.yml` file may help in diagnosing the issue. Paste the contents of the plugin's config file here: (feel free to omit your Discord token for security reasons)
```yaml
# How long the hunters should get blindness and slowness when the match starts.
headStartDuration: 25
compassEnabledInNether: false
sendUsageData: false
enableDiscord: true
discordToken: x
ip: servas.ggs.gg
processDiscordCommands: false
discordServerId: 773923702651092992
musicChannelId: 773923702651092996
hunterRoleId: XXXXX
runnerRoleId: XXXXX
spectatorRoleId: XXXXX
uuid: 100c7e93-1313-4290-8941-aca8dea14701
```

## Server Logs
Please check your Server Console for any warnings or errors created by the plugin. If any are any, please copy and paste them here:
```
None.
```

## Anything else?
Nope.

**Comment by yummypasta** [comment]
Sorry about this! I thought I had fixed it in a previous release but I'll look at it again.

**Comment by yummypasta** [comment]
Fixed in b66b6e02c77dc1671e2b6df1fe7ea49ce07273dc. Please download [release 1.3.2](https://github.com/yummypasta/MinecraftManhunt/releases/tag/1.3.2) for a bug fix!
